packages/styling/variables/colors.scss

```diff
@@ -48,3 +48,11 @@ $dso-success-bg-color:  $lime-20;
 $dso-info-bg-color:     #e1ecf7;
 $dso-warning-bg-color:  #f8f6cc;
 $dso-danger-bg-color:   #f5d8dc;
+
+////
+
+$brand-primary: $bosgroen;
+$brand-success: $dso-success-color;
```

Comment:
`$brand-success` samen met -info, -warning en -danger zijn aliassen van $dso-success-color. Waar we dat voorheen nodig hadden omdat we de variable in de Bootstrap namespace moesten aansturen, is dat nu niet nodig: Ik stel voor dat we één variant gebruiken.

Ik denk dat we de variant zonder `$dso-` prefix moeten aanhouden, want de Bootstrap variant bestaat nog steeds en wordt nog steeds gebruikt.